nodebuilder/tests/swamp/swamp.go

```diff
@@ -181,9 +183,15 @@ func (s *Swamp) setupGenesis() {
 
 	host, port, err := net.SplitHostPort(s.ClientContext.GRPCClient.Target())
 	require.NoError(s.t, err)
-	client := core.NewClient(host, port)
-	require.NoError(s.t, client.Start())
-	fetcher, err := core.NewBlockFetcher(client)
+	addr := net.JoinHostPort(host, port)
+	con, err := coremodule.NewGRPCClient(
+		addr,
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	require.NoError(s.t, err)
+	con.Connect()
```

Comment:
Did you try without? In tests it might not be neccesary to call it, as it connects lazily.